prql-compiler/src/lib.rs

```diff
@@ -105,18 +105,17 @@ pub static PRQL_VERSION: Lazy<Version> =
 /// Use the prql compiler to convert a PRQL string to SQLite dialect
 ///
 /// ```
-/// use prql_compiler::compile;
-/// use prql_compiler::sql;
-///
+/// use prql_compiler::{compile, sql};
 ///
 /// let prql = "from employees | select [name,age] ";
 /// let opt = sql::Options {
-///     format: true,
+///     format: false,
 ///     dialect: Some(sql::Dialect::SQLite),
-///     signature_comment: true
+///     signature_comment: false
 /// };
 /// let sql = compile(&prql, Some(opt)).unwrap();
-/// println!("PRQL: {}\nSQLite: {}", prql, sql);
+/// println!("PRQL: {}\nSQLite: {}", prql, &sql);
+/// # assert_eq!("SELECT name, age FROM employees", sql)
```

Comment:
Is this commented out intentionally?

Reply:
The `#` comment just [hides that line](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#hiding-portions-of-the-example) in the generated rustdocs or if you were to include the example it in another file. I added it when I was trying to just include this example directly in the book, but I couldn't get it to work properly in with the `{#rustdoc_include ...}` so it isn't needed anymore. I'll update.